tests/puppeteer/pages/FO/home.js

```diff
@@ -6,7 +6,7 @@ module.exports = class Home extends CommonPage {
 
     // Selectors for home page
     this.logoHomePage = '#_desktop_logo';
-    this.productImg = '#content article:nth-child(%NUMBER) img';
+    this.productImg = '#content div:nth-child(%NUMBER) article img';
```

Comment:
```suggestion
    this.productImg = '.products div:nth-child(%NUMBER) article img';
```
Maybe starting from list of products is better than content which contain carousel and some other things ?

Reply:
I think it's better too

Reply:
Yes, it's true, but I think that it's better to use an id than a class name in a selector
I propose to add an id for the products bloc, what do you think?

Reply:
```suggestion
    this.productImg = '#content .products div:nth-child(%NUMBER) article img';
```